types/core/types.d.ts

```diff
@@ -2,6 +2,7 @@ export namespace boolean {
     function coerce(val: any): boolean;
 }
 export namespace number {
+    function coerce(val: any): any;
```

Comment:
Why there is duplicate declaration?

Reply:
I believe it's a bug with the TypeScript version used because when I try the latest it generates only one signature. But anyway, it still works because it interprets them as overloads.